etc/kayobe/ansible/smartmon-tools.yml

```diff
@@ -2,12 +2,13 @@
 - hosts: overcloud
 
   tasks:
-    - name: Ensure smartmon-tools and nvme-cli is installed
+    - name: Ensure smartmon-tools, jq, nvme-cli and cron/cronie is installed
```

Comment:
This is why I usually just put "packages" or "package dependencies" :)

Reply:
Ah. Will bear in mind for next time